Lesson3/src/HomeworkLesson3.java

```diff
@@ -0,0 +1,105 @@
+import java.util.Random;
+import java.util.Scanner;
+
+public class HomeworkLesson3 {
+
+    public static void main(String[] args) {
+        System.out.println(operation(1));
+        System.out.println(operation(0));
+        foobar(6);
+        foobar(10);
+        foobar(15);
+        printMatrix();
+        printPrimeNumbers();
+    }
+
+    /**
+     * Method should perform some operation with int "number" based on some conditions:
+     * - if number is positive - increase it by 1
+     * - if number is negative - decrease it by 2
+     * - if number is equal to zero - replace it with 10
+     * return number value after operation
+     */
+    public static int operation(int number) {
+        if (number > 0) {
+            number++;
+        } else if (number < 0) {
+            number = number - 2;
+        } else {
+            number = 10;
+        }
+        return number;
+    }
+
+
+    /**
+     * Method should print different strings in console based on some conditions:
+     * - if remainder of the division number by 3 is zero - print "foo" (example of number - 6)
+     * - if remainder of the division number by 5 is zero - print "bar" (example of number - 10)
+     * - if remainder of the division number by both 3 and 5 is zero 0  and - print "foobar" (example of number - 15)
+     */
+    public static void foobar(int number) {
+        if (((double) number % 3 == 0) && ((double) number % 5 == 0)) {
```

Comment:
you don't need to cast number to double here. % works with int and double in the same way